### ISSUE TITLE
dffram should allow pointing to a local openlane install

### ISSUE THREAD

**Issue by proppy** [AUCOHL/DFFRAM#119]
https://github.com/Cloud-V/DFFRAM/blob/main/docs/Usage.md#using-dffram seems to suggest `docker` is only required to run `placeram` but in practice almost all the steps depends on the `openlane` function `run_docker` invocation: https://github.com/Cloud-V/DFFRAM/blob/main/dffram.py#L77

It would be nice to optionally allow users to point to a local ~~openroad~~ openlane install in order to allow them to run it in environment where docker is not supported (i.e: https://colab.research.google.com/).

**Comment by donn** [comment]
It's a little bit less straightforward than that since we also do rely on a number of scripts inside the OpenLane container and we also rely on some OpenLane utilities such as magic for sign-off.

That said, assuming someone is willing to download an OpenLane repo and point to it + compile magic themselves, I'm not opposed to the idea.

**Comment by proppy** [comment]
Makes sense, so I should probably rename the issue to 'pointing to a local openlane' install :)

**Comment by donn** [comment]
Yeah, that's a lot more doable. I'll look into it.

**Comment by proppy** [comment]
Looks like this was fixed with #143?
